### PR TITLE
LPS-179523 Avoid using some taglibs

### PR DESCRIPTION
**Pull request by ling-alan-huang**
https://issues.liferay.com/browse/LPS-179523

### CODE REVIEW

**Comment by ling-alan-huang** [comment]
Hi @ealonso, we finished your SF request, and wanted to see if it's what you want.
You can test on your end by the following steps:

1. Fetch this PR
2. Run `../../../gradlew deploy` in `modules/util/source-formatter`
3. Commit your changes which contine old taglibs
4. Run `ant format-source-current-branch` in `portal-impl`

File: source-formatter.properties [hunk 1]
```diff
@@ -547,6 +547,19 @@
 
     source.check.IllegalImportsCheck.enforceCookiesManagerUtil=true
 
+    source.check.IllegalImportsCheck.replacedTaglibs=\
```
Comment (ling-alan-huang) [inline source-formatter.properties:550]: @ealonso Just feel free to add old taglibs here.

File: source-formatter.properties [hunk 2]
```diff
@@ -547,6 +547,19 @@
...
+        com.liferay.taglib.aui.ColTag->com.liferay.frontend.taglib.clay.servlet.taglib.ColTag
+
+    source.check.IllegalTaglibsCheck.ftlReplacedTaglibs=\
```
Comment (ling-alan-huang) [inline source-formatter.properties:553]: @ealonso Just feel free to add old taglibs here.

**Comment by ling-alan-huang** [comment]
ci:test:relevant

File: source-formatter.properties [hunk 3]
```diff
@@ -547,6 +547,19 @@
 
     source.check.IllegalImportsCheck.enforceCookiesManagerUtil=true
 
+    source.check.IllegalImportsCheck.replacedTaglibs=\
+        com.liferay.taglib.aui.ColTag->com.liferay.frontend.taglib.clay.servlet.taglib.ColTag
```
Comment (ling-alan-huang) [inline source-formatter.properties:551]: @ealonso Just feel free to add old taglib class names.

File: source-formatter.properties (hunk 2 above)
Comment (ling-alan-huang) [reply source-formatter.properties:553]: Btw, once on old tag can be replaced in jsp, it should also be replaced in ftl, am I saying that correctly?
I am thinking if we should make a pattern for both jsp and ftl.

Comment (ealonso) [reply source-formatter.properties:553]: Yes, it is correct. We shouldn't have old taglib nor in jsp and nor in ftl

**Comment by ealonso** [comment]
Hey @ling-alan-huang,

Everything looks good for me, I've already tested it and it is working as expected. 

I would like to update the different properties, with the following values:

```
source.check.IllegalImportsCheck.replacedTaglibs=\
        com.liferay.taglib.aui.IconTag->com.liferay.frontend.taglib.clay.servlet.taglib.IconTag

source.check.IllegalTaglibsCheck.ftlReplacedTaglibs=\
      @liferay_aui.a->@clay.a,\
      @liferay_aui.alert->@clay.alert,\
      @liferay_aui.col->@clay.col,\
      @liferay_aui.icon->@clay.icon,\
      @liferay_aui.row->@clay.row,\
      @liferay_aui["a"]->@clay["a"],\
      @liferay_aui["alert"]->@clay["alert"],\
      @liferay_aui["col"]->@clay["col"],\
      @liferay_aui["icon"]->@clay["icon"],\
      @liferay_aui["row"]->@clay["row"]

  source.check.IllegalTaglibsCheck.jspReplacedTaglibs=\
      aui:a->clay:link,\
      aui:alert->clay:alert,\
      aui:col->clay:col,\
      aui:icon->clay:icon,\
      aui:row->clay:row,\
      aui:workflow-status->liferay-portal-workflow:status,\
      liferay-ui:icon->clay:icon,\
      liferay-ui:icon-deactivate->clay:icon,\
      liferay-ui:icon-delete->clay:icon,\
      liferay-ui:icon-help->clay:icon,\
      liferay-ui:icon-menu->clay:dropdown-actions,\
      liferay-ui:tabs->clay:tabs
```

Should I include it and send it to Brian? What should be the next step? 

Thanks!!

**Comment by ling-alan-huang** [comment]
@ealonso Sure, you can send the PR with your properties to Brian.
Once I have another better way for this SF request I will let you know.

Btw, why `jspReplacedTaglibs` has `aui:workflow-status->liferay-portal-workflow:status`,
but `ftlReplacedTaglibs` doesn't have it.

**Comment by ealonso** [comment]
@ling-alan-huang related with the difference between jsp and ftl, it is intended, I'm only adding the used taglibs in ftl, the same for java.

**Comment by ealonso** [comment]
See: https://github.com/brianchandotcom/liferay-portal/pull/133873

**Pull request closed without merging**